workflows/service_common_test.go

```diff
@@ -14,7 +14,7 @@ func TestServiceLoader_FromConfig(t *testing.T) {
 	assert := assert.New(t)
 
 	ctx := new(common.Context)
-	ctx.Config.Repo.Name = "myrepo"
+	ctx.Config.Repo.Name = "foo/myrepo"
```

Comment:
This seems incorrect, shouldn't be prefixing the org name since that has it's own field.